static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-usa-cta.js

```diff
@@ -0,0 +1,220 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic-equal-buttons.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features',
+    'lodash/arrays/intersection'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpicEqualButtons,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             ajax,
+             commercialFeatures,
+             intersection) {
+
+    return function () {
+
+        this.id = 'ContributionsEpicUsaCta';
+        this.start = '2016-11-15';
+        this.expiry = '2016-11-22';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test just contributions vs contributions or membership in the US';
+        this.showForSensitive = false;
+        this.audience = 1;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Impressions to number of contributions / supporter sign ups';
+        this.audienceCriteria = 'Just the US';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'We prove or disprove our hypothesis that just offering contributions will result in an overall boost in money taken in the USA';
+        this.canRun = function () {
+            var includedKeywordIds = [
+                'us-news/us-elections-2016',
+                'us-news/us-politics'
+            ];
+
+            var excludedKeywordIds = ['music/leonard-cohen'];
+
+            var hasKeywordsMatch = function() {
+                var pageKeywords = config.page.keywordIds;
+                if (typeof(pageKeywords) != 'undefined') {
```

Comment:
Yep. In this case, this if statement isn't necessary as `"".split(',')` = `['']`.